### ISSUE TITLE
About the Datasets

### ISSUE THREAD

**Issue by DualProblem** [scy-phy/ICS-Evasion-Attacks#2]
Dear Authors,

Thanks for your great efforts on this work! I really appreciated it!

I have some doubts related to the datasets.

For wadi dataset,
1) I noted that in "list_of_considered_features.csv", there are 85 features that have been considered. But in "Attacked_Model/WADI.ipynb", the input dimension is 82. So what are the other 3 ignored dimensions?
2) I requested the wadi dataset from iTrust, the feature dimension should be 123. So what is the rule that you use to filter some features? 
3) I also noted that in "Attacked_Model/WADI.ipynb",  
`df_test_01 = pd.read_csv(data_path + 'attacks_october_clean_with_label.csv')`
So I wonder what this dataset is. From iTrust, I can only got "WADI_14days_new.csv" and "WADI_attackdataLABLE.csv"

For BATADAL, below are some statistics of the BATADAL dataset I requested from iTrust:
#Train: 8761 (BATADAL_dataset03.csv)
#Test: 4177 (BATADAL_dataset04.csv)

And the data points in this dataset is collected per hour. But in "Data/BATADAL/train_dataset.csv", the number of data points is 48106, and the data seems to be collected per 0.25 hour. The same difference can also be found in  "Data/BATADAL/test_dataset_1.csv" and Data/BATADAL/test_dataset_2.csv"

Could you please tell me the source of these datasets?

Looking forward to your kind reply!

**Comment by axoaxel** [comment]
Hi, thanks for the interest in our work. 
This repository is based on the ACSAC 2020 research paper "Constrained Concealment Attacks against Reconstruction-based Anomaly Detectors in Industrial Control Systems" https://publications.cispa.saarland/3226/

WADI dataset:
1. I just pushed  in the repository the updated list of features, there were 3 features extra (that were later ignored by the scripts). Thanks for pointing it out.
2. In our paper you can find the related details about the considered sensors in section 5.2.
3. We processed the original dataset "WADI_14days_new.csv" to add the labels as a column called ATT_FLAG

BATADAL dataset:
The information related to the BATADAL dataset (contained in this repository) can be found in the paper in section 5.1.

Best,
Alessandro
